### PR TITLE
Move Strings to json

### PR DESCRIPTION
**Pull request by 6543**
move strings from template to json

TODO:
 * [x] one string.json
 * [x] in asset folder
 * [x] use strings from preview
 * [x] add all strings to json
 * [x] no raw strings in ui templates
 * [x] no raw strings in JS/TS

### CODE REVIEW

**Comment by 6543** [comment]
one question: should we just use 1 "strings.json" ?

File: src/strings/header.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "header": {
+        "docTitle": "Dokument Titel:",
+        "org": "Organisation:",
```
Comment (6543) [inline src/strings/header.json:4]: organisation

File: src/app/search/edit-search-results/edit-search-results.component.html [hunk 2]
```diff
@@ -165,46 +147,31 @@
...
-            mat-icon-button
-            (click)="addNewPPhrase()"
-            (mouseenter)="addPPhraseHover = true"
-            (mouseleave)="addPPhraseHover = false"
-            matTooltip="P-Satz hinzufügen"
-            matTooltipPosition="right"
-          >
+          <a mat-icon-button (click)="addNewPPhrase()" (mouseenter)="addPPhraseHover = true"
+            (mouseleave)="addPPhraseHover = false" [matTooltip]="strings.substance.addPPhrase"
+            matTooltipPosition="right">
```
Comment (Epsilon02) [inline src/app/search/edit-search-results/edit-search-results.component.html:174]: Was this refactoring by Prettier?
For me it is less readable than before

File: src/app/search/edit-search-results/edit-search-results.component.html [hunk 3]
```diff
@@ -213,18 +180,13 @@
...
-            class="symbol undragable"
-            *ngFor="let key of symbolKeys"
-            [ngClass]="isSymbolActive(key) ? '' : 'grayscale'"
-            (click)="toggleSymbol(key)"
-            [src]="sanitizeImage(key)"
-            alt="{{ key }}"
-          />
+          <img class="symbol undragable" *ngFor="let key of symbolKeys"
+            [ngClass]="isSymbolActive(key) ? '' : 'grayscale'" (click)="toggleSymbol(key)" [src]="sanitizeImage(key)"
+            alt="{{ key }}" />
```
Comment (Epsilon02) [inline src/app/search/edit-search-results/edit-search-results.component.html:189]: It is less readable than before

File: src/app/search/edit-search-results/edit-search-results.component.html [hunk 4]
```diff
@@ -165,46 +147,31 @@
...
+              <textarea matInput formControlName="pPhrase" cdkTextareaAutosize cdkAutosizeMinRows="1"
+                cdkAutosizeMaxRows="5"></textarea>
+              <button matSuffix mat-icon-button (click)="removePhrase(i, pPhrases)"
                 (mouseenter)="pPhrases.at(i).get('hover')?.patchValue(true)"
-                (mouseleave)="pPhrases.at(i).get('hover')?.patchValue(false)"
-                matTooltip="P-Satz entfernen"
-                matTooltipPosition="left"
-              >
+                (mouseleave)="pPhrases.at(i).get('hover')?.patchValue(false)" [matTooltip]="strings.substance.rmPPhrase"
+                matTooltipPosition="left">
```
Comment (Epsilon02) [inline src/app/search/edit-search-results/edit-search-results.component.html:164]: Less readable than before

**Review by Epsilon02 — COMMENTED** [review]
Some refactorings in HTMLs are in my opinion less readable.
I marked a few but not all.
Was this refactoring done by Prettier?

File: src/app/search/edit-search-results/edit-search-results.component.html (hunk 2 above)
Comment (6543) [reply src/app/search/edit-search-results/edit-search-results.component.html:174]: > refactoring by Prettier

yes ... we need to make sure Prettier works on all systems as expected

File: src/app/search/edit-search-results/edit-search-results.component.html [hunk 5]
```diff
@@ -2,70 +2,70 @@
   <mat-dialog-content class="grid unselectable">
     <mat-card class="mini-grid">
       <mat-form-field>
-        <mat-label>Name</mat-label>
+        <mat-label>{{ strings.substance.name }}</mat-label>
         <input matInput formControlName="name" />
-        <mat-error *ngIf="this.form.get('name')?.get('value')?.invalid">Name benötigt</mat-error>
+        <mat-error *ngIf="this.form.get('name')?.get('value')?.invalid">{{ strings.substance.needName }}</mat-error>
```
Comment (crapStone) [inline src/app/search/edit-search-results/edit-search-results.component.html:7]:
```suggestion
        <mat-error *ngIf="this.form.get('name')?.get('value')?.invalid">{{ strings.substance.needsName }}</mat-error>
```

File: src/app/search/edit-search-results/edit-search-results.component.html [hunk 6]
```diff
@@ -89,12 +89,12 @@
       </div>
       <div formGroupName="amount">
         <mat-form-field>
-          <mat-label>Menge</mat-label>
+          <mat-label>{{ strings.substance.amount }}</mat-label>
           <input matInput formControlName="value" />
-          <mat-error *ngIf="this.form.get('amount')?.get('value')?.invalid">Keine Zahl</mat-error>
+          <mat-error *ngIf="this.form.get('amount')?.get('value')?.invalid">{{ strings.substance.noNumber }}</mat-error>
```
Comment (crapStone) [inline src/app/search/edit-search-results/edit-search-results.component.html:94]:
```suggestion
          <mat-error *ngIf="this.form.get('amount')?.get('value')?.invalid">{{ strings.substance.notANumber }}</mat-error>
```

File: src/app/search/edit-search-results/edit-search-results.component.html [hunk 7]
```diff
@@ -108,7 +108,7 @@
           </mat-select>
         </mat-form-field>
         <mat-form-field *ngIf="customUnitVisible">
-          <mat-label>Eigene Einheit</mat-label>
+          <mat-label>{{ strings.substance.ownUnit }}</mat-label>
```
Comment (crapStone) [inline src/app/search/edit-search-results/edit-search-results.component.html:111]:
```suggestion
          <mat-label>{{ strings.substance.customUnit }}</mat-label>
```

File: src/app/search/edit-search-results/edit-search-results.component.html [hunk 8]
```diff
@@ -139,7 +139,7 @@
                 (click)="removePhrase(i, hPhrases)"
                 (mouseenter)="hPhrases.at(i).get('hover')?.patchValue(true)"
                 (mouseleave)="hPhrases.at(i).get('hover')?.patchValue(false)"
-                matTooltip="P-Satz entfernen"
+                [matTooltip]="strings.substance.rmHPhrase"
```
Comment (crapStone) [inline src/app/search/edit-search-results/edit-search-results.component.html:142]:
```suggestion
                [matTooltip]="strings.substance.removeHPhrase"
```

File: src/app/search/edit-search-results/edit-search-results.component.html [hunk 9]
```diff
@@ -187,7 +187,7 @@
                 (click)="removePhrase(i, pPhrases)"
                 (mouseenter)="pPhrases.at(i).get('hover')?.patchValue(true)"
                 (mouseleave)="pPhrases.at(i).get('hover')?.patchValue(false)"
-                matTooltip="P-Satz entfernen"
+                [matTooltip]="strings.substance.rmPPhrase"
```
Comment (crapStone) [inline src/app/search/edit-search-results/edit-search-results.component.html:190]:
```suggestion
                [matTooltip]="strings.substance.removePPhrase"
```

File: src/app/search/edit-search-results/edit-search-results.component.html [hunk 10]
```diff
@@ -213,8 +213,8 @@
     </div>
 
     <mat-card class="center" formArrayName="symbols">
-      <p>Zum entfernen oder hinzufügen auf das Symbol klicken</p>
-      <p>Grau: nicht gewählt / Farbig: Gewählt</p>
+      <p>{{ strings.substance.ghsSymbolsAddRmInfo }}</p>
```
Comment (crapStone) [inline src/app/search/edit-search-results/edit-search-results.component.html:216]:
```suggestion
      <p>{{ strings.substance.ghsSymbolsAddOrRemoveInfo }}</p>
```

File: src/app/search/edit-search-results/edit-search-results.component.html [hunk 11]
```diff
@@ -234,7 +234,7 @@
     </div>
   </mat-dialog-content>
   <div mat-dialog-actions>
-    <button mat-button (click)="close()">Abbrechen</button>
-    <button mat-button type="submit">Speichern</button>
+    <button mat-button (click)="close()">{{ strings.base.cancle }}</button>
```
Comment (crapStone) [inline src/app/search/edit-search-results/edit-search-results.component.html:237]:
```suggestion
    <button mat-button (click)="close()">{{ strings.base.cancel }}</button>
```

File: src/app/search/search-dialog/search-dialog.component.html [hunk 12]
```diff
@@ -1,8 +1,8 @@
-<h2>Suchergebnisse</h2>
+<h2>{{ strings.search.resultHeader }}</h2>
```
Comment (crapStone) [inline src/app/search/search-dialog/search-dialog.component.html:1]:
```suggestion
<h2>{{ strings.search.resultsHeader }}</h2>
```

File: src/app/search/search-dialog/search-dialog.component.html [hunk 13]
```diff
@@ -20,10 +20,10 @@ <h2>Suchergebnisse</h2>
...
 </ng-template>
 <ng-template #empty>
-  <p class="viewport empty"><b>Keine Ergebnisse</b></p>
+  <p class="viewport empty"><b>{{ strings.search.noResult }}</b></p>
 </ng-template>
 <mat-divider class="divider"></mat-divider>
 <div mat-dialog-actions>
-  <button mat-button (click)="close(true)">Abbrechen</button>
-  <button mat-button (click)="close(false)">Ok</button>
+  <button mat-button (click)="close(true)">{{ strings.base.cancle }}</button>
```
Comment (crapStone) [inline src/app/search/search-dialog/search-dialog.component.html:27]:
```suggestion
  <button mat-button (click)="close(true)">{{ strings.base.cancel }}</button>
```

File: src/app/search/search-dialog/search-dialog.component.ts [hunk 14]
```diff
@@ -40,10 +44,10 @@ export class SearchDialogComponent implements OnInit {
       this.searchResults = response;
       this.searchFinished = true;
     },
-    (err) => {
-      logger.error('loading search results failed:', err);
-      this.alertService.error('Laden der Suchergebnisse fehlgeschlagen!');
-    });
+      (err) => {
+        logger.error('loading search results failed:', err);
+        this.alertService.error(strings.error.loadSearchResult);
```
Comment (crapStone) [inline src/app/search/search-dialog/search-dialog.component.ts:49]:
```suggestion
        this.alertService.error(strings.error.loadSearchResults);
```

File: src/app/search/search.component.html [hunk 15]
```diff
@@ -3,21 +3,21 @@
     <!-- TODO make this to one way databinding -->
     <app-selected-search (triggerSearch)="openDialog()" class="width-100"></app-selected-search>
   </div>
-  <button class="button unselectable" mat-flat-button color="primary" (click)="openDialog()">Suchen</button>
+  <button class="button unselectable" mat-flat-button color="primary" (click)="openDialog()">{{ strings.search.search }}</button>
```
Comment (crapStone) [inline src/app/search/search.component.html:6]:
```suggestion
  <button class="button unselectable" mat-flat-button color="primary" (click)="openDialog()">{{ strings.search.searchButton }}</button>
```

File: src/app/search/search.component.html [hunk 16]
```diff
@@ -3,21 +3,21 @@
...
-        <mat-header-cell *matHeaderCellDef> Name</mat-header-cell>
+        <mat-header-cell *matHeaderCellDef> {{ strings.substance.name }}</mat-header-cell>
         <mat-cell *matCellDef="let element">
           {{ element.name.modifiedData ? element.name.modifiedData : element.name.originalData }}
         </mat-cell>
       </ng-container>
 
       <ng-container matColumnDef="cas" [sticky]="true">
-        <mat-header-cell *matHeaderCellDef> CAS</mat-header-cell>
+        <mat-header-cell *matHeaderCellDef> {{ strings.substance.casNr }}</mat-header-cell>
```
Comment (crapStone) [inline src/app/search/search.component.html:20]:
```suggestion
        <mat-header-cell *matHeaderCellDef> {{ strings.substance.casNumber }}</mat-header-cell>
```

File: src/app/search/search.component.html [hunk 17]
```diff
@@ -38,7 +38,7 @@
             <button
               mat-icon-button
               (click)="removeSubstance($event, element)"
-              matTooltip="Substanz entfernen"
+              [matTooltip]="strings.search.rmSubstance"
```
Comment (crapStone) [inline src/app/search/search.component.html:41]:
```suggestion
              [matTooltip]="strings.search.removeSubstance"
```

File: src/app/search/selected-search/selected-search.component.html [hunk 18]
```diff
@@ -3,7 +3,7 @@
     <div *ngFor="let control of selections.controls; let i = index">
       <div class="flex" [formGroupName]="i">
         <mat-form-field class="select">
-          <mat-label>Suchoption</mat-label>
+          <mat-label>{{strings.search.option}}</mat-label>
```
Comment (crapStone) [inline src/app/search/selected-search/selected-search.component.html:6]:
```suggestion
          <mat-label>{{ strings.search.option }}</mat-label>
```

File: src/app/search/selected-search/selected-search.component.html [hunk 19]
```diff
@@ -35,7 +35,7 @@
               (click)="removeSearchOption(i)"
               (mouseenter)="selections.at(i).get('hover')?.patchValue(true)"
               (mouseleave)="selections.at(i).get('hover')?.patchValue(false)"
-              matTooltip="Suchoption entfernen"
+              [matTooltip]="strings.search.rmOption"
```
Comment (crapStone) [inline src/app/search/selected-search/selected-search.component.html:38]:
```suggestion
              [matTooltip]="strings.search.removeOption"
```

File: src/app/search/search.component.ts [hunk 20]
```diff
@@ -72,20 +76,19 @@ export class SearchComponent implements OnInit {
             if (
               cas && this.globals.substanceDataSubject.getValue().some(s => cas === this.modifiedOrOriginal(s.cas))
             ) {
-              // TODO i18n
-              this.alertService.error('Substanz mit selber CAS Nummer existiert bereits!');
               logger.warning('substance with same cas number already present:', cas);
+              this.alertService.error(strings.error.substanceWithCASSExist);
```
Comment (crapStone) [inline src/app/search/search.component.ts:80]:
```suggestion
              this.alertService.error(strings.error.substanceWithCASExist);
```

File: src/app/preview/preview.component.html [hunk 21]
```diff
@@ -115,15 +106,15 @@ <h2 class="mat-h2 preview-text"><b>Dies is eine Vorschau. Die tatsächliche, gen
       </tr>
       <tr>
         <td class="top height-2cm">
-          {{ descriptions.signStudent }}
+          {{ strings.descriptions.signStudent }}
```
Comment (crapStone) [inline src/app/preview/preview.component.html:109]:
```suggestion
          {{ strings.descriptions.signatureFieldStudent }}
```

File: src/app/preview/preview.component.html [hunk 22]
```diff
@@ -115,15 +106,15 @@ <h2 class="mat-h2 preview-text"><b>Dies is eine Vorschau. Die tatsächliche, gen
...
         </td>
         <td class="top height-2cm">
-          {{ descriptions.signAssistant }}
+          {{ strings.descriptions.signAssistant }}
```
Comment (crapStone) [inline src/app/preview/preview.component.html:112]:
```suggestion
          {{ strings.descriptions.signatureFieldAssistant }}
```

**Review by crapStone — CHANGES_REQUESTED** [review]

File: src/assets/strings.json [hunk 23]
```diff
@@ -0,0 +1,103 @@
+{
+  "strings": {
+    "base": {
+      "addElement": "Eintrag hinzufügen",
+      "removeElement": "Eintrag entfernen",
+      "print": "Print",
+      "cancle": "Abbrechen",
```
Comment (crapStone) [inline src/assets/strings.json:7]:
```suggestion
      "cancel": "Abbrechen",
```

File: src/assets/strings.json [hunk 24]
```diff
@@ -0,0 +1,103 @@
...
+      "search": "Suche",
+      "rmSubstance": "Substanz entfernen",
+      "rmOption": "Suchoption entfernen",
+      "addOption": "Suchoption hinzufügen",
+      "resultHeader": "Suchergebnisse",
+      "noResult": "Keine Ergebnisse"
+    },
+    "substance": {
+      "name": "Name",
+      "needName": "Name benötigt",
```
Comment (crapStone) [inline src/assets/strings.json:39]:
```suggestion
      "needsName": "Name benötigt",
```

File: src/assets/strings.json [hunk 25]
```diff
@@ -0,0 +1,103 @@
...
+      "waterHazardClass": "Wassergefährdungsklasse",
+      "signalWord": "Signalwort",
+      "molarMass": "Molare Masse",
+      "unit": "Einheit",
+      "lethalDose": "Lethale Dosis",
+      "mak": "MAK",
+      "meltingPoint": "Schmelzpunkt",
+      "boilingPoint": "Siedepunkt",
+      "amount": "Menge",
+      "noNumber": "Keine Zahl",
```
Comment (crapStone) [inline src/assets/strings.json:51]:
```suggestion
      "notANumber": "Keine Zahl",
```

File: src/assets/strings.json [hunk 26]
```diff
@@ -0,0 +1,103 @@
...
+      "ownUnit": "Eigene Einheit",
```
Comment (crapStone) [inline src/assets/strings.json:52]:
```suggestion
      "customUnit": "Eigene Einheit",
```

File: src/assets/strings.json [hunk 27]
```diff
@@ -0,0 +1,103 @@
...
+      "hPhrases": "H-Sätze",
+      "noHPhrase": "Ungültige H-Nummer",
+      "rmHPhrase": "P-Satz entfernen",
```
Comment (crapStone) [inline src/assets/strings.json:55]:
```suggestion
      "removeHPhrase": "P-Satz entfernen",
```

File: src/assets/strings.json [hunk 28]
```diff
@@ -0,0 +1,103 @@
...
+      "addHPhrase": "H-Satz hinzufügen",
+      "pPhrases": "P-Sätze",
+      "noPPhrase": "Ungültige P-Nummer",
+      "rmPPhrase": "P-Satz entfernen",
```
Comment (crapStone) [inline src/assets/strings.json:59]:
```suggestion
      "removePPhrase": "P-Satz entfernen",
```

File: src/assets/strings.json [hunk 29]
```diff
@@ -0,0 +1,103 @@
...
+      "addPPhrase": "P-Satz hinzufügen",
+      "ghsSymbolsAddRmInfo": "Zum entfernen oder hinzufügen auf das Symbol klicken",
```
Comment (crapStone) [inline src/assets/strings.json:61]:
```suggestion
      "ghsSymbolsAddOrRemoveInfo": "Zum entfernen oder hinzufügen auf das Symbol klicken",
```

File: src/assets/strings.json [hunk 30]
```diff
@@ -0,0 +1,103 @@
...
+      "preparation": "Herzustellendes Präparat:"
+    },
+    "search": {
+      "exact": "Exakte Suche",
+      "option": "Suchoption:",
+      "search": "Suche",
+      "rmSubstance": "Substanz entfernen",
+      "rmOption": "Suchoption entfernen",
+      "addOption": "Suchoption hinzufügen",
+      "resultHeader": "Suchergebnisse",
```
Comment (crapStone) [inline src/assets/strings.json:34]:
```suggestion
      "resultsHeader": "Suchergebnisse",
```

File: src/assets/strings.json [hunk 31]
```diff
@@ -0,0 +1,103 @@
...
+    },
+    "error": {
+      "substanceWithCASSExist": "Substanz mit selber CAS Nummer existiert bereits!",
+      "substanceLoadData": "Laden der Daten der Substanz fehlgeschlagen!",
+      "editSubstance": "Bearbeiten der Substanz fehlgeschlagen!",
+      "configLoad": "Laden der Konfigurationsdatei fehlgeschlagen!",
+      "configSave": "Speichern der Konfigurationsdatei fehlgeschlagen!",
+      "getHazardSymbols": "Laden der Gefahrensymbole fehlgeschlagen!",
+      "loadSearchSuggestions": "Laden der Suchvorschläge fehlgeschlagen!",
+      "loadSearchResult": "Laden der Suchergebnisse fehlgeschlagen!",
```
Comment (crapStone) [inline src/assets/strings.json:91]:
```suggestion
      "loadSearchResults": "Laden der Suchergebnisse fehlgeschlagen!",
```

File: src/assets/strings.json [hunk 32]
```diff
@@ -0,0 +1,103 @@
...
+      "labCourse": "Lab/Kurs:",
+      "name": "Name:",
+      "place": "Platz:",
+      "assistant": "Assistent/in:",
+      "preparation": "Herzustellendes Präparat:"
+    },
+    "search": {
+      "exact": "Exakte Suche",
+      "option": "Suchoption:",
+      "search": "Suche",
```
Comment (crapStone) [inline src/assets/strings.json:30]:
```suggestion
      "searchButton": "Suche",
```

File: src/assets/strings.json [hunk 33]
```diff
@@ -0,0 +1,103 @@
...
+      "rmSubstance": "Substanz entfernen",
+      "rmOption": "Suchoption entfernen",
+      "addOption": "Suchoption hinzufügen",
+      "resultHeader": "Suchergebnisse",
+      "noResult": "Keine Ergebnisse"
+    },
+    "substance": {
+      "name": "Name",
+      "needName": "Name benötigt",
+      "casNr": "CAS Nummer",
```
Comment (crapStone) [inline src/assets/strings.json:40]:
```suggestion
      "casNumber": "CAS Nummer",
```

File: src/assets/strings.json [hunk 34]
```diff
@@ -0,0 +1,103 @@
...
+      "name": "Name:",
+      "place": "Platz:",
+      "assistant": "Assistent/in:",
+      "preparation": "Herzustellendes Präparat:"
+    },
+    "search": {
+      "exact": "Exakte Suche",
+      "option": "Suchoption:",
+      "search": "Suche",
+      "rmSubstance": "Substanz entfernen",
```
Comment (crapStone) [inline src/assets/strings.json:31]:
```suggestion
      "removeSubstance": "Substanz entfernen",
```

File: src/assets/strings.json [hunk 35]
```diff
@@ -0,0 +1,103 @@
...
+      "rmOption": "Suchoption entfernen",
```
Comment (crapStone) [inline src/assets/strings.json:32]:
```suggestion
      "removeOption": "Suchoption entfernen",
```

File: src/assets/strings.json [hunk 36]
```diff
@@ -0,0 +1,103 @@
...
+      "tableMolarMassHeader": "MG\n[g/mol]",
+      "tableBoilingAndMeltingHeader": "Sdp.\nSmp.",
+      "tableGHSSymbolHeader": "GHS-Symbol(e)",
+      "tableHPPhrasesHeader": "Nummern der H/P-Sätze",
+      "tableMAKHeader": "MAK, LD50\nWGK",
+      "tableAmountHeader": "für Ansatz benötigt",
+      "tableHPPhrasesTitle": "Wortlaut der wesentlichen oben genannten H- und P-Sätze:"
+    },
+    "error": {
+      "substanceWithCASSExist": "Substanz mit selber CAS Nummer existiert bereits!",
```
Comment (crapStone) [inline src/assets/strings.json:84]:
```suggestion
      "substanceWithCASExist": "Substanz mit selber CAS Nummer existiert bereits!",
```

File: src/assets/strings.json [hunk 37]
```diff
@@ -0,0 +1,103 @@
...
+      "addPPhrase": "P-Satz hinzufügen",
+      "ghsSymbolsAddRmInfo": "Zum entfernen oder hinzufügen auf das Symbol klicken",
+      "ghsSymbolsColorInfo": "Grau: nicht gewählt / Farbig: Gewählt"
+    },
+    "descriptions": {
+      "humanAndEnvironmentDanger": "Gefahren für Mensch und Umwelt, die von den Ausgangsmaterialien bzw. dem(n) Produkt ausgehen, soweit sie nicht durch genannte Angaben abgedeckt sind (z.B. krebserregend, fruchtschädigend, hautresorptiv):",
+      "rulesOfConduct": "Schutzmaßnahmen und Verhaltensregeln:",
+      "inCaseOfDanger": "Verhalten im Gefahrenfall, Erste-Hilfe-Maßnahmen (gegebenfalls Kopie der entsprechenden Literaturstelle beiheften):",
+      "disposal": "Entsorgung:",
+      "signStudent": "Hiermit verpflichte ich mich, den Versuch gemäß den in dieser Betriebsanweisung aufgeführten Sicherheitsvorschriften durchzuführen.",
```
Comment (crapStone) [inline src/assets/strings.json:69]:
```suggestion
      "signatureFieldStudent": "Hiermit verpflichte ich mich, den Versuch gemäß den in dieser Betriebsanweisung aufgeführten Sicherheitsvorschriften durchzuführen.",
```

File: src/assets/strings.json [hunk 38]
```diff
@@ -0,0 +1,103 @@
...
+      "signAssistant": "Präparat zur Synthese mit den auf der Vorderseite berechneten Chemikalienmengen freigegeben."
```
Comment (crapStone) [inline src/assets/strings.json:70]:
```suggestion
      "signatureFieldAssistant": "Präparat zur Synthese mit den auf der Vorderseite berechneten Chemikalienmengen freigegeben."
```

**Review by crapStone — CHANGES_REQUESTED** [review]

**Review by crapStone — APPROVED** [review]

**Pull request merged**